### ISSUE TITLE
Too much ARIA

### ISSUE THREAD

**Issue by LJWatson** [alex-ju/accessible-loading-spinner#1]
The ```<div>``` has too much ARIA:
```
<div id="loader" class="govuk-c-loader" role="progressbar" aria-busy="true" aria-atomic="false" aria-label="Loading, please wait." aria-labelledby="loading-label" aria-live="polite" tabindex="0">
```

* There is no need for the `aria-live` and `aria-atomic` attributes if you use the `progressbar` role.
- The `aria-busy` attribute should be used on the container of the content that is loading, not on the load spinner itself (because in some cases it causes the element to be removed from the accessibility tree until the attribute is removed/set to `false`.
* You cannot use `aria-label` and `aria-labelledby` at the same time, and there is no need to give this thing an accessible name.

Suggest using a simple progress bar design pattern:
```
<div role="progressbar" aria-valuemin="0" aria-valuemax="100" aria-valuenow="30" aria-label="Loading">
```

The `aria-valuenow` attribute should be updated dynamically to reflect the current state of progress. It is the value of this attribute that screen readers will announce.

If a simple number announcement is insufficient, use the `aria-valuetext` attribute to provide a human readable equivalent of the `aria-valuenow` attribute. For example:
```
<div role="progressbar" aria-valuemin="0" aria-valuemax="100" aria-valuenow="30" aria-valuetext="30%" aria-label="Loading">
```

The value of the `aria-label` attribute will be announced in conjunction with the progress. So a screen reader should announce something like "Loading 30%".
